diffsync/diff.py

```diff
@@ -33,6 +33,7 @@ def __init__(self):
 
         `self.children[group][unique_id] == DiffElement(...)`
         """
+        self.models_processed = 0
```

Comment:
`models` could be a bit confusing, at least to me. What about `objects`?

Reply:
The `DiffSyncDiffer` already had the logic to count the processed models and the name used is `models_prossed`, so I thought it would be good to use the same name for consistency, but I have no problem renaming it
https://github.com/ubaumann/diffsync/blob/689c917428970061346b1715a18575ee0c04bb4d/diffsync/helpers.py#L72